pkg/controller/cyclenoderequest/transitioner/transitioner.go

```diff
@@ -91,6 +91,7 @@ func (t *CycleNodeRequestTransitioner) transitionFuncs() map[v1.CycleNodeRequest
 		v1.CycleNodeRequestWaitingTermination: t.transitionWaitingTermination,
 		v1.CycleNodeRequestFailed:             t.transitionFailed,
 		v1.CycleNodeRequestSuccessful:         t.transitionSuccessful,
+		v1.CycleNodeRequestSkipped:			   t.transitionSkipped,
```

Comment:
nit: align this with the others

Reply:
Goland + tab weirdness, and indeed git status shows wrong indentation while it's aligned ok in the editor. Should be ok now.